src/debug/ee/amd64/amd64walker.cpp

```diff
@@ -744,7 +744,7 @@ void NativeWalker::DecodeInstructionForPatchSkip(const BYTE *address, Instructio
             }
 
             // RET
-            if ((lowNibble == 0x2) && (lowNibble == 0x3))
+            if ((lowNibble == 0x2) || (lowNibble == 0x3))
```

Comment:
@jasonwilliams200OK Nope, this code is inside `case 0xC:` of `switch(highNibble)`. Code you suggest would be dead too.